lisa/search_space.py

```diff
@@ -0,0 +1,464 @@
+from abc import abstractmethod
+from dataclasses import dataclass, field
+from typing import Any, Iterable, List, Optional, Set, TypeVar, Union
+
+from dataclasses_json import LetterCase, dataclass_json  # type: ignore
+
+from lisa.util import LisaException
+
+_MAX = 2147483647
```

Comment:
Why this value? Shouldn't it be `sys.maxsize`?

Reply:
updated